packages/eslint-config-custom/index.js

```diff
@@ -18,6 +18,15 @@ module.exports = {
     '@typescript-eslint/no-unnecessary-condition': ['error', { allowConstantLoopConditions: true }],
     'react/react-in-jsx-scope': 'off',
     'react/prop-types': 'off',
+    // Catches untyped let declarations
+    'no-restricted-syntax': [
+      'error',
+      {
+        selector:
+          "VariableDeclaration[kind = 'let'] > VariableDeclarator[init = null]:not([id.typeAnnotation])",
```

Comment:
OMG. I need to learn how to write these custom rules. I've been wanting to make an import sorting order linting rule forever.

Reply:
Maybe it's simpler than we think: https://eslint.org/docs/latest/rules/sort-imports